### PR TITLE
Update urls in "about" to avoid redirects

### PR DESCRIPTION
**Pull request by duramato**

### CODE REVIEW

File: about.pas [hunk 1]
```diff
@@ -212,7 +212,7 @@ procedure TCheckVersionThread.Execute;
 procedure TAboutForm.imgSynapseClick(Sender: TObject);
 begin
   AppBusy;
-  OpenURL('http://synapse.ararat.cz');
+  OpenURL('http://synapse.ararat.cz/doku.php');
```
Comment (PeterDaveHello) [inline about.pas]: It's a 302 redirection not 301, I prefer not to touch it.

File: about.pas [hunk 2]
```diff
@@ -251,7 +251,7 @@ procedure TAboutForm.imgDonateClick(Sender: TObject);
 procedure TAboutForm.imgLazarusClick(Sender: TObject);
 begin
   AppBusy;
-  OpenURL('http://www.lazarus.freepascal.org');
+  OpenURL('https://www.lazarus-ide.org');
```
Comment (PeterDaveHello) [inline about.pas]: This one seems to be good 👍

File: about.pas (hunk 1 above)
Comment (duramato) [inline about.pas]: Changed it back @PeterDaveHello

**Comment by PeterDaveHello** [comment]
Landed on v5.8.3 here: https://github.com/leonsoft-kras/transmisson-remote-gui/releases/tag/v5.8.3, thanks for your contribution.

**Pull request merged**